### PR TITLE
attempt to add all rewrites to nginx

### PR DESCRIPTION
**Pull request by neurosnap**

### CODE REVIEW

**Comment by almathew** [comment]
I haven't looked at this at all, but I'm hitting `nginx: [emerg] invalid parameter "permenant" in /etc/nginx/nginx.conf:29` when building the Docker container

File: nginx.conf [hunk 1]
```diff
@@ -25,6 +25,436 @@ http {
     add_header "X-Frame-Options" "DENY" always;
     add_header "X-Content-Type-Options" "nosniff";
     add_header "X-Permitted-Cross-Domain-Policies" "master-only";
+
+    rewrite ^/documentation/*$ http://documentation-production.s3-website-us-east-1.amazonaws.com/documentation/* permenant;
```
Comment (fancyremarker) [inline nginx.conf:29]: This should be `permanent` not `permenant`, right?

Comment (almathew) [reply nginx.conf:29]: Yes. There's also a problem with line 456 (I think we need to remove the `302!`). I'm working on those changes now but for some reason nginx still isn't responding to requests

**Comment by almathew** [comment]
Pushed the changes. I think this works, although like I mentioned, I have an issue that I think is Docker-specific that's preventing me from confirming

**Comment by neurosnap** [comment]
pushed a change with some misplaced `$` in the rewrite rules

**Review by fancyremarker — APPROVED** [review]

**Pull request merged**